concordium-sdk/src/main/java/com/concordium/sdk/responses/poolstatus/PassiveDelegationStatus.java

```diff
@@ -0,0 +1,53 @@
+package com.concordium.sdk.responses.poolstatus;
+
+import com.concordium.sdk.responses.accountinfo.CommissionRates;
+import com.concordium.sdk.serializing.JsonMapper;
+import com.concordium.sdk.transactions.CCDAmount;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import concordium.ConcordiumP2PRpc;
+import lombok.Data;
+import lombok.Getter;
+import lombok.ToString;
+
+@Data
+@Getter
+@ToString
+public class PassiveDelegationStatus extends PoolStatus {
```

Comment:
Make fields final please.
I don't see the need for @Getter @ToString when @Data is used.

Reply:
done

Reply:
done